Project.toml

```
@@ -1,26 +1,21 @@
 name = "RegisterMismatchCuda"
 uuid = "ee0d8d85-fa18-576c-9601-66ebb12862d9"
 authors = ["Tim Holy <tim.holy@gmail.com>"]
-version = "0.3.1"
+version = "0.4.1"
```

```suggestion
version = "0.3.2"
```
This actually isn't a breaking change, in the sense that any user of this package doesn't have to change a single line of their code (I hope).